.github/workflows/boolean_benchmark.yml

```diff
@@ -92,6 +92,16 @@ jobs:
           --name-suffix avx512 \
           --append-results
 
+      - name: Measure key sizes
+        run: |
+          RUSTFLAGS="-C target-cpu=native" cargo +nightly --example boolean_key_sizes --features="x86_64-unix boolean internal-keycache"
```

Comment:
can you create a target in the makefile perhaps ?

Reply:
We could also create target for benchmarks no ?

Reply:
ah yes ! don't know why I did not think of this !

Reply:
By the way, do we need the nightly Rust toolchain to be used for the tests?

Reply:
I believe benchmarks with AVX support needs nightly. And it does seems that the Makefile support installation of a nightly tool chain. Am I wrong ?

Reply:
avx-512 support needs nighlty yes

Reply:
the check toolchain is always a nighlty one so you can use that one as toolchain, if you want we can have a quick call